src/components/table/table.tsx

```diff
@@ -385,9 +385,12 @@ export class Table {
     }
 
     render() {
+        const pagination =
+            this.totalRows <= this.pageSize ? 'has-no-pagination' : null;
+
         return (
             <div id="tabulator-container">
-                <div id="tabulator-table" />
+                <div id="tabulator-table" class={pagination} />
```

Comment:
If you use Bef's suggestion for `const pagination` above, I think you need this too:

```suggestion
                <div id="tabulator-table" class={{pagination}} />
```